.github/workflows/linux.yml

```diff
@@ -29,11 +29,15 @@ jobs:
         sudo add-apt-repository ppa:ginggs/deal.ii-9.3.2-backports
         sudo apt-get update
         sudo apt-get install -yq --no-install-recommends libdeal.ii-dev
+        #TODO: Remove this when github actions ships cmake 3.26.1 or younger
+        cd /opt && wget https://github.com/Kitware/CMake/releases/download/v3.26.1/cmake-3.26.1-linux-x86_64.tar.gz
```

Comment:
Do you want to delete the tar file as well?

Reply:
I guess I could, but it doesnt matter very much. This container is deleted after running anyway, so saving space is not helping anything.